posthog-core/src/types.ts

```diff
@@ -1,32 +1,34 @@
 export type PosthogCoreOptions = {
```

Comment:
Since its a major version with breaking changes, should we rename types from `Posthog` to `PostHog`?

Reply:
good thinking - updated everywhere